src/main/java/microservices/userServiceBasic/api/response/UserResponse.java

```diff
@@ -0,0 +1,29 @@
+package microservices.userServiceBasic.api.response;
+
+
+import microservices.userServiceBasic.api.model.UserDetails;
+
+import java.util.UUID;
+
+public class UserResponse extends UserDetails {
+
+	private boolean responseSuccessful;
```

Comment:
This response should be always successful. If something goes wrong it should return an ErrorResponse. For now Spring will handle this and it returns a generic Response with the exception thrown by the application. Later we'll build an Error handling.